### PR TITLE
MY 1, 2, 6, 8, 12

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
TEI version of MY 008, minor changes in a few other mss.

### CODE REVIEW

File: ES/ESmy008.xml [hunk 1]
```diff
@@ -30,328 +30,751 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                                 s.l.: 6, stub after 2
+                               </item>
+
+                              <item xml:id="q20" n="19">
                                  <dim unit="leaf">5</dim>
                                  <locus from="140r" to="144v"/>
-                                 <locus target="#13"/>
-                                 <locus target="#19"/> XIX(5; s.l. 1, no stub/fols. 140r-144v)&#xD;
-                                 f. 13 is misplaced; it should be placed after fol. 19.</item>
+                                 </item>
```
Comment (eu-genia) [inline ES/ESmy008.xml:403]: here the info on s.l. disappeared?
add
s.l. 1, no stub

File: ES/ESmy008.xml [hunk 2]
```diff
@@ -30,328 +30,751 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
                   <origin>
                      <origPlace>
-                        <placeName ref="INS0039MY">Dabra Maʿṣo / MY</placeName>
                      </origPlace>
-                     <origDate notBefore="1382" notAfter="1412">From the time of King Dāwit II (r.
-                        1382-1412).&#xD; The name of King Zarʾa Yāʿqob (r. 1344-68) is mentioned in
-                        Additio 2 (<locus target="#3ra">3ra</locus>).</origDate>
+                     <origDate notBefore="1370" notAfter="1440">Late fourteenth/early fifteenth century,
+                       probably 1382-1412</origDate>
+                       The manuscript could be copied by the same scribe as
```
Comment (eu-genia) [inline ES/ESmy008.xml:769]: may have been copied

File: ES/ESmy008.xml [hunk 3]
```diff
@@ -30,328 +30,751 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                       Mss <ref type="mss" corresp="ESmy002">MY-002</ref> and  <ref type="mss" corresp="ESmy001">MY-001</ref>,
+                       donated together with them by <persName ref="PRS13561TasabkaMadhen">Tasabka Madḫən</persName>,
```
Comment (eu-genia) [inline ES/ESmy008.xml:771]: add role="donor"?

File: ES/ESmy008.xml [hunk 4]
```diff
@@ -397,14 +827,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                  </person>
+                  <person>
+                    <persName role="scribe" ref="PRS13638HabtaMaryam">Habta Māryām</persName>
                </person>
             </listPerson>
          </particDesc>
          <textClass>
             <keywords scheme="#ethioauthlist">
                <term key="NewTestament"/>
+               <term key="ChristianLiterature"/>
```
Comment (eu-genia) [inline ES/ESmy008.xml:840]: `<term key="GoldenGospel"/>`

File: ES/ESmy008.xml [hunk 5]
```diff
@@ -414,11 +847,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="SA" when="2011-03-02">catalogued</change>
          <change who="DN" when="2015-09-11">last edited</change>
          <change who="PL" when="2016-05-10">transformed from mycore to TEI P5</change>
+         <change who="MK" when="2021-07-20">Ethiopic passages suplied</change>
```
Comment (eu-genia) [inline ES/ESmy008.xml:850]: supplied

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ES/ESmy008.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy008.xml:771]: ok

File: ES/ESmy008.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy008.xml:840]: ok

File: ES/ESmy008.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply ES/ESmy008.xml:850]: ok

**Review by eu-genia — APPROVED** [review]

**Pull request merged**